### DISCUSSION TITLE
Where do fonts in tailwindui themes come from?

### DISCUSSION THREAD

**Discussion by djsegal** [tailwindlabs/tailwindcss#1504] [General]
I looked at the svg paths for a few icons and the collections you linked (i.e. entypo, zondicons, heroicons), but couldn't find any exact matches.

Did you custom build the icons for the themes? Or are they from another source?

**Comment by adamwathan** [comment]
I'm guessing "fonts" in the title is a typo and you meant "icons" based on the actual issue content, if so the icons are from here:

https://github.com/refactoringui/heroicons

It's mentioned in the docs here:

https://www.notion.so/Tailwind-UI-Documentation-f9083ed0e2694690ac89253e88afb2b6#17f63634951e4351adf07d51bfc56ef6
